docs/file-structure.md

```diff
@@ -26,6 +26,18 @@ This directory should be placed in the plugin's root, next to the `public.php` f
     - https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/invoice-csv-export/src/public
     - https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/plugins/invoice-csv-export/src/templates/form.php#L10
 
+### public/admin-zone.js
+*Note: This feature is available since UNMS 1.1.0-beta2*
+
+If this file exists, it will be automatically loaded on every admin zone page in UCRM.
+Use the `window.location` object to determine the URL.
```

Comment:
this is JS feature and as such belongs to JS documentation, not to CRM plugin docs
if we had some SDK to handle this, then it would belong here
```suggestion
```